src/node_jsvmapi.cc

```diff
@@ -496,6 +492,48 @@ namespace v8impl {
 
 }  // end of namespace v8impl
 
+// Intercepts the Node-V8 module registration callback. Converts parameters to NAPI equivalents
+// and then calls the registration callback specified by the NAPI module.
+void napi_module_register_cb(v8::Local<v8::Object> exports,
+                             v8::Local<v8::Value> module,
+                             v8::Local<v8::Context> context,
+                             void* priv) {
+  napi_module* mod = static_cast<napi_module*>(priv);
```

Comment:
is there any error checking needed here? Can any of the calls below fail? if so, what happens?